sql/size_utils.sql

```diff
@@ -92,7 +92,9 @@ $BODY$
             0::BIGINT AS compressed_toast_size,
             0::BIGINT AS compressed_heap_size
         FROM
-            _timescaledb_catalog.hypertable
+            _timescaledb_catalog.hypertable ht
+            INNER JOIN pg_class c ON relname = ht.table_name
+            AND c.relnamespace IN (SELECT oid FROM pg_namespace WHERE nspname = schema_name_in)
```

Comment:
Why use a sub-query instead of a simple JOIN? The plan will not change but the code will be more readable using explicit JOINs.

Reply:
Yeah make sense. Converted sub-query to join.